contracts/extensions/sequential-governance/CoreGovernance.sol

```diff
@@ -65,29 +65,31 @@ abstract contract CoreGovernance is SignatureConsumer, VoteStatusConsumer, Chain
     _setProposalExpiryDuration(_expiryDuration);
   }
 
-  /**
-   * @dev Creates new round voting for the proposal `_proposalHash` of chain `_chainId`.
-   */
-  function _createVotingRound(
-    uint256 _chainId,
-    bytes32 _proposalHash,
-    uint256 _expiryTimestamp
-  ) internal returns (uint256 _round) {
+  function _calculateVotingRoundAndDeleteExpiry(uint256 _chainId) internal returns (uint256 _round) {
     _round = round[_chainId];
-
     // Skip checking for the first ever round
     if (_round == 0) {
       _round = round[_chainId] = 1;
     } else {
       ProposalVote storage _latestProposalVote = vote[_chainId][_round];
       bool _isExpired = _tryDeleteExpiredVotingRound(_latestProposalVote);
-      // Skip increase round number if the latest round is expired, allow the vote to be overridden
+      // Skip increasing round number if the latest round is expired, allow the vote to be overridden
       if (!_isExpired) {
         require(_latestProposalVote.status != VoteStatus.Pending, "CoreGovernance: current proposal is not completed");
         _round = ++round[_chainId];
       }
     }
+  }
 
+  /**
+   * @dev Creates new round voting for the proposal `_proposalHash` of chain `_chainId`.
+   */
+  function _createVotingRound(
```

Comment:
I guess this function is more like assigning arguments (`hash`, `expiryTimestamp`) to the voting round. Consider renaming it.